File: boat-spring-boot-core/src/test/java/test/xyz/srclab/spring/boot/autoconfigure/AutoConfigureTest.java
package test.xyz.srclab.spring.boot.autoconfigure;

import org.springframework.test.context.ContextConfiguration;
import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
import org.testng.annotations.Test;
import xyz.srclab.spring.boot.autoconfigure.CoreAutoConfiguration;

@ContextConfiguration(classes = {CoreAutoConfiguration.class})
public class AutoConfigureTest extends AbstractTestNGSpringContextTests {

    @Test
    public void testAutoConfigure() {

    }
}

File: spring-boot-autoconfigure/easy-spring-boot-autoconfigure-test/src/test/kotlin/other/autoconfigure/test/AutoConfigureTest.kt
package other.autoconfigure.test

import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.testng.AbstractTestNGSpringContextTests
import org.testng.annotations.Test

@SpringBootTest(classes = [Config::class])
class AutoConfigureTest : AbstractTestNGSpringContextTests() {

    @Test
    fun testAutoConfigure() {
    }
}